.github/workflows/detect-breaking-changes-report.yml

```diff
@@ -129,13 +134,15 @@ jobs:
           }
       env:
         SLACK_WEBHOOK_URL: ${{ secrets.SLACK_LEVITATE_WEBHOOK_URL }}
+        HAS_SECRETS: ${{ (github.repository == "grafana/grafana" || secrets.SLACK_LEVITATE_WEBHOOK_URL != '') || '' }}
```

Comment:
@torkelo here's an example of using an `if` to check for a specific repository.

If you don't want things to run on other repositories, you'd just use more of this logic and less of the rest.